### PR TITLE
[New] test cases for pdf-exporter

### PR DESCRIPTION
**Pull request by Srinivas9933**

### CODE REVIEW

File: app/javascript/components/pdf-exporter/component.spec.js [hunk 1]
```diff
@@ -76,20 +74,19 @@ describe("<PdfExporter />", () => {
...
+    setupMockFormComponent(PdfExporter, { props });
+    expect(screen.getAllByText(/exports.printed/i)).toHaveLength(2);
   });
 
   it("renders Logos", () => {
-    const { component } = setupMockFormComponent(PdfExporter, { props });
+    setupMockFormComponent(PdfExporter, { props });
 
-    expect(component.find(Logos)).to.have.lengthOf(2);
+    expect(screen.getAllByText((content, element) => element.tagName.toLowerCase() === "svg")).toHaveLength(1);
```
Comment (aespinoza-quoin) [inline app/javascript/components/pdf-exporter/component.spec.js:84]: `content` variable is not being used, add a prefix or use regular expresion

File: app/javascript/components/pdf-exporter/components/logos/component.spec.js [hunk 2]
```diff
@@ -21,14 +21,14 @@ describe("<Logos />", () => {
...
-    expect(component.find(Logos)).to.have.lengthOf(1);
+    expect(screen.getByText(/exports.printed/i)).toBeInTheDocument();
   });
 
   it("renders img", () => {
-    const { component } = setupMountedComponent(Logos, props);
+    mountedComponent(<Logos {...props} />);
 
-    expect(component.find("img")).to.have.lengthOf(2);
+    expect(screen.getAllByText((content, element) => element.tagName.toLowerCase() === "img")).toHaveLength(2);
```
Comment (aespinoza-quoin) [inline app/javascript/components/pdf-exporter/components/logos/component.spec.js:32]: `content` variable is not being used, add a prefix or use regular expresion

**Review by aespinoza-quoin — CHANGES_REQUESTED** [review]

File: app/javascript/components/pdf-exporter/component.spec.js (hunk 1 above)
Comment (Srinivas9933) [reply app/javascript/components/pdf-exporter/component.spec.js]: Done

File: app/javascript/components/pdf-exporter/components/logos/component.spec.js (hunk 2 above)
Comment (Srinivas9933) [reply app/javascript/components/pdf-exporter/components/logos/component.spec.js]: Done

**Pull request merged**